Winium/Winium.StoreApps.InnerServer/Commands/ClickCommand.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Winium.StoreApps.InnerServer.Commands
+{
+    #region
+
+    using Windows.UI.Xaml.Automation.Peers;
+    using Windows.UI.Xaml.Automation.Provider;
+    using Windows.UI.Xaml.Controls;
+    using Common;
+
+    #endregion
+
+    internal class ClickCommand : CommandBase
+    {
+        #region Public Properties
+
+        public string ElementId { private get; set; }
+
+        #endregion
+
+        #region Public Methods and Operators
+
+        protected override string DoImpl()
+        {
+            var element = this.Automator.ElementsRegistry.GetRegisteredElement(this.ElementId);
+            Button button = element.Element as Button;
+            if (button != null)
+            {
+                ButtonAutomationPeer peer = new ButtonAutomationPeer(button);
+                IInvokeProvider invokeProv = peer.GetPattern(PatternInterface.Invoke) as IInvokeProvider;
+                if (invokeProv != null)
+                {
+                    invokeProv.Invoke();
+                    return this.JsonResponse(ResponseStatus.Success, "");
+                }
+
+                return this.JsonResponse(ResponseStatus.UnknownError, "Failed to create invocation provider");
+            }
+
+            return this.JsonResponse(ResponseStatus.UnknownError, "Element is not a button");
```

Comment:
I'll check. It's possible we only get "Winium IDs" here. ( long-ish number with a dash followed by a short number ) I'm not sure what the point of these are, they are causing a few issues and it would be nice to just get our own element ID back. Though I fear changing that would mean a significant change to Winium itself